### PR TITLE
Skill Cape Update Part 1

### PR DESCRIPTION
**Pull request by TheFriggenSawbones**
ITS HERE

ADDS THE CRAFTING SKILL CAPE
and the various items need to craft it

http://puu.sh/qnN6J/91f51f9d3e.png

CRAFTED FROM,

A PLATE
BLANKET
GAVEL
CHISEL
WIRE
AND A SCREWDRIVER.

### CODE REVIEW

**Comment by clusterfack** [comment]
No skill capes

**Comment by TheFriggenSawbones** [comment]
WAIT LETS TEST THIS A BIT BUADS IS TELLING ME THINGS I SWEAR IT WORKED MY STOLEN CODE IS FLAWLESS

**Comment by TheFriggenSawbones** [comment]
ok there we go everythings gucci

File: code/game/objects/items/weapons/courtroom.dm [hunk 1]
```diff
@@ -39,3 +41,23 @@
...
+		return
+	if(istype(W,/obj/item/weapon/chisel) && !stage)
+		stage = 1
+		to_chat(user,"<span class='notice'>You add \the [W] to \the [src].</span>")
+		qdel(W)
+		icon_state = "gavelhammer_1"
+	if(istype(W,/obj/item/stack/cable_coil) && stage == 1)
+		var/obj/item/stack/cable_coil/C = W
+		if(C.amount <= 4)
+			return
```
Comment (ghost) [inline code/game/objects/items/weapons/courtroom.dm:58]: I have to ask, was this all stolen from the ghetto space suit?

Comment (TheFriggenSawbones) [inline code/game/objects/items/weapons/courtroom.dm:58]: hell yeah

**Comment by ghost** [comment]
Add HP cape I can wear in medbay, add Quest Cape for prize at the end of a Gateway.

**Comment by kernikov** [comment]
fresh memes

**Comment by carshalash** [comment]
Give cooking cape.

**Comment by Wizbing** [comment]
Instead of being craftable, why not make them an alternative to trophies for people who excel at their job?

Would be a fun update, we don't have any runescape references, and would give people reason to actually try to do their job for a week or so.

construct _cape_detail

MOMMI with cape slots when?

**Comment by TheFriggenSawbones** [comment]
Thats what I was thinking.

For starters, Crafting would be awarded from crafting. Arthur wanted a mining cape to drop randomly from Xenoarch/mining. I've been trying to think of other ways to trigger or get the capes.

**Comment by Wizbing** [comment]
honk

500 hours in ms honk

**Comment by Kurfursten** [comment]
@ComicIronic Used to always make me use defines for crafting steps.

File: code/game/objects/items/weapons/courtroom.dm [hunk 2]
```diff
@@ -39,3 +41,23 @@
 			cooldown = world.time
 		return 1
 	return ..()
+
+/obj/item/weapon/gavelhammer/attackby(obj/item/W,mob/user)
```
Comment (PJB3005) [inline code/game/objects/items/weapons/courtroom.dm:45]: This should use construction datums.

File: code/modules/clothing/back/cape.dm [hunk 3]
```diff
@@ -3,4 +3,57 @@
...
+	var/canstage = 1
+	var/stage = 0
+
+/obj/item/clothing/back/craftingsigil
+	name = "Crafting sigil"
+	desc = "An ornate icon of the fourth age."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "gavelhammer_2"
+
+/obj/item/clothing/back/skillcapebase/attackby(obj/item/W,mob/user)
```
Comment (PJB3005) [inline code/modules/clothing/back/cape.dm:30]: Definitely gonna need construction datums.

**Comment by ComicIronic** [comment]
Defines for maintaining existing code, construction datums for new code!

**Comment by icantthinkofanameritenow** [comment]
RuneScape a shit

**Comment by SkowronX** [comment]
Run escape.

**Comment by clusterfack** [comment]
Skill capes are fucking dumb mate

**Comment by gbasood** [comment]
Okay but what the fuck is the point of this
Is it just to have craftable capes? Because that's the most dumb thing ever.

**Comment by TheFriggenSawbones** [comment]
its to have SKILL CAPES for doing things that REQUIRE SKILL

we have a magic cape, soon we'll have a mining cape.

Crafting is the only craftable one.

**Comment by SonixApache** [comment]
delete this

**Comment by gbasood** [comment]
nah that's dumb
this isn't an MMO, especially not Runescape

**Comment by ScrambledSteak** [comment]
local men refuse to have fun

**Comment by icantthinkofanameritenow** [comment]
@TheFriggenSawbones 
Nothing in this game requires skill, or at least no skill that deserves a cape.

**Comment by AutismSpider** [comment]
>skill cape

What the fuck for?

**Comment by TheFriggenSawbones** [comment]
skill man what else

**Comment by AutismSpider** [comment]
What skill?

You got the wrong game for this my man.

**Comment by TheFriggenSawbones** [comment]
Magic, Crafting, Mining

all valuable skills.

**Comment by icantthinkofanameritenow** [comment]
First, this game doesn't have "skills" that can be raised via ingame mechanics -those capes were basically a big sign saying "I wasted my life grinding up a semi-useless skill/hired a Chinese man in a sweatshop to play my game for me, bow down before my excess of spare time". So what if they all have magic, mining, and crafting? So does Minecraft, and I don't see any skill capes in that game.  

Second, even if we did have such a system it's supposed to be awarded, not actually made by the player.

**Comment by kol1th** [comment]
> conflicts

:joy:

**Comment by ihadtoregisterforthis** [comment]
Icons

**Comment by PJB3005** [comment]
http://84.195.252.227/dmi

**Comment by SonixApache** [comment]
why is this still open?

**Comment by PJB3005** [comment]
Democracy, clearly.

**Comment by gbasood** [comment]
Honestly this is bad entirely because this kind of flavor has always been doled out by admins
If it weren't skillcapes and you could get them via merch or cargo crate then I'd be okay with it maybe

**Comment by ghost** [comment]
Okay, so if we were to have skillcapes depending on jobs, we'd need

> Skillcape of clowning
> Skillcape of silence
> Skillcape of Cooking
> Skillcape of Herblore
> Skillcape of Crafting
> Skillcape of power
> Skillcape of construction
> Skillcape of Toxins (For atmos techs of toxins scientists)
> Skillcape of Slime
> Skillcape of robotics
> Skillcape of alchemy
> Skillcape of HP (For MDs that don't just use cryoclone)
> Skillcape of viral
> Skillcape of prayer
> Skillcape of questing
> Skillcape of cleaning
> Skillcape of delivery

**Comment by gbasood** [comment]
And ways to make those capes and also REASONS to make those capes other than watching an entire station running around LARPing inside SS13 while malf waits patiently for their cycles

**Comment by icantthinkofanameritenow** [comment]
And ways where any form of actual skill would be involved in earning them. Right now you can be surprisingly good at everything just by following the wiki's instructions; plus, the skill capes require a skill _system_ to be in place - the closest thing to that is what Bay has and even there it's an RP guideline rather than a game mechanic. 

I'd suggest trying to copy LifeWeb's skill system since it's closest to what would make sense here, but it's closed source. And...well, it's LifeWeb.

**Comment by PJB3005** [comment]
> skills
> 
> ss13.

Fuck no.

**Comment by icantthinkofanameritenow** [comment]
@PJB3005
Exactly.

**Comment by ComicIronic** [comment]
Good skill systems for mechanical games do not exist, unless we want to pull XCOM levels of RNG.

**Comment by Venseer** [comment]
I would be fine if it was just the capes as a cosmetic.

**Comment by SkowronX** [comment]
still conflicts

**Comment by AlphaBuffalo** [comment]
I like the idea of just making the capes trophies that are awarded by admins

**Comment by gbasood** [comment]
^

**Comment by icantthinkofanameritenow** [comment]
Don't we already have trophies for that?

**Comment by AlphaBuffalo** [comment]
I don't think we have any cape trophies

**Comment by PJB3005** [comment]
Yeah but in that case they DEFINITELY need to be able to be worn by MoMMIs because how else will I get my autism cape?

**Comment by AutismSpider** [comment]
@PJB3005 Need backpack slot for mommis then

**Comment by ghost** [comment]
MoMMIs just need a crown, or a CE helmet with the wooper

**Comment by AlphaBuffalo** [comment]
Mommis with backpacks doesn't sound too bad for this, most of the time they drag a locker or something with them

**Comment by SonixApache** [comment]
delete this

**Comment by AlphaBuffalo** [comment]
I mean what can a mommi with a backpack do? :^)

**Comment by gbasood** [comment]
neglected and dead PR
either re-open this if you find a way to address the issues or make a new one I guess

**Pull request closed without merging**